### DISCUSSION TITLE
MS-MPI installation failed error code 50. A version of MPI that is the same or newer is already installed

### DISCUSSION THREAD

**Discussion by avcortiz** [blueCFD/Core#210] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/82894737/115424524-c8407580-a1cc-11eb-895e-e6ea7106f8a7.JPG)
Help!

**Comment by wyldckat** [comment, marked as answer]
Quick answer: Unfortunately we currently don't have a way to be certain of which MS-MPI version is already or not installed in Windows.

When this message appears, as mentioned in the error message, there are two possible reasons for it to have happened:

1. Either MS-MPI 7.1 has already been installed, in which situation you don't need to worry about it, since it's already installed.
2. Or a more recent MS-MPI version is already installed, in which situation... it can affect how blueCFD-Core works. But it should not affect anything else in your Windows installation.

The second situation is the only one of concern for blueCFD-Core, but it should work as intended. If it doesn't please let us know!

**Reply by wyldckat** [reply]
Please also check the discussion #204, which has additional instructions on how to use an older MSMPI version, if a newer one is already installed in Windows.
